runners/flink/src/test/java/org/apache/beam/runners/flink/streaming/WindowDoFnOperatorTest.java

```diff
@@ -148,7 +148,9 @@ public void testRestore() throws Exception {
         emptyMap(),
         emptyList(),
         PipelineOptionsFactory.as(FlinkPipelineOptions.class),
-        VarLongCoder.of()
+        VarLongCoder.of(),
+        null /* key selector */ // TODO use proper key selector
```

Comment:
Maybe create a JIRA for this one if you don't plan to address it in this PR.

Reply:
apperently we don't need a key selector here so I'll remove the TODO